spec/person_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../classes/person'
+
+describe Person do
+  person = Person.new(28, true, 'Sajeel')
+  describe 'Person Information' do
+    it 'should have a correct name of Sajeel' do
+      expect(person.correct_name).to eql('Sajeel')
+    end
+    it 'should have an age of 28' do
+      expect(person.age).to eql(28)
+    end
+    it 'should be able to use services' do
+      expect(person.can_use_services?).to eql(true)
+    end
+  end
+end
```

Comment:
- [ Required ] Please consider adding a `unit test` for the `add_rentals` method in class `Person`, the project requirements state that you create unit tests for all the methods implemented in each class, currently there is no unit test for the `add_rentals` method in class `Person`. 

![Screenshot from 2023-01-11 23-48-51](https://user-images.githubusercontent.com/16392046/211915406-613cd2f4-7214-4005-9b0a-2ca7e2b57c4c.png)